android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportEvent.kt

```diff
@@ -24,7 +24,7 @@ import org.smartregister.fhircore.quest.ui.shared.models.MeasureReportPatientVie
 
 sealed class MeasureReportEvent {
   data class OnSelectMeasure(
-    val measureReportConfig: MeasureReportConfig,
+    val measureReportConfig: List<MeasureReportConfig>?,
```

Comment:
We can default to an `emptyList` instead of `null` unless `null` has a meaning for this use case.